lib/dockerregistry/inventory.go

```diff
@@ -52,9 +52,7 @@ func getSrcRegistry(rcs []RegistryContext) (*RegistryContext, error) {
 // MakeSyncContext creates a SyncContext.
 func MakeSyncContext(
 	manifestPath string,
```

Comment:
Can this also be removed?

Reply:
Yes, but it will be removed in a forthcoming change (when #64 is implemented and it no longer makes sense to include this in the logs).